apps/condo/domains/marketplace/schema/MarketPriceScope.test.js

```diff
@@ -32,6 +34,17 @@ describe('MarketPriceScope', () => {
     describe('Accesses', () => {
         describe('admin', () => {
             test('can create', async () => {
+                const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                    canReadMarketItems: true,
+                    canReadMarketItemPrices: true,
+                    canReadMarketPriceScopes: true,
+                    canManageMarketItemPrices: true,
+                    canManageMarketPriceScopes: true,
+                })
+                await createTestOrganizationEmployee(admin, organization, admin.user, role)
```

Comment:
Why do you need to add a role for admin? `isAdmin=true` has access everywhere.

Reply:
this needs to be removed) I added it to create test data for the front)